assets/src/components/editing/slateUtils.tsx

```diff
@@ -74,7 +74,7 @@ export const isTopLevel = (editor: Editor) => {
   const [...nodes] = Editor.nodes(editor, {
     match: (n) => {
       if (!Element.isElement(n)) return false;
-      return schema[n.type].isTopLevel;
+      return schema[n.type]?.isTopLevel || false;
```

Comment:
This would crash if you had an element in your document that a definition didn't exist for.

Probably never/rarely happen in real life, but me switching between branches with different elements available and using the same document hit it.